### PR TITLE
Feat: Created `genshin` app

### LINKED ISSUE #76: [Feat]: Create Genshin module
### My suggestion

Very simple, just create new django app and register it correctly.

### Possible alternatives

_No response_

### PR DESCRIPTION
**Pull request by Sanielia**
Last pr was somehow messed up…

### CODE REVIEW

**Review by Norbiros — CHANGES_REQUESTED** [review]
Add simple main genshin page and register it in panel.

**Review by hexxcatee — DISMISSED** [review]
apruf

File: genshin/templates/genshin/index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{% extends "core/base.html" %}
+
+{% block title %}Genshin Module{% endblock %}
```
Comment (Norbiros) [inline genshin/templates/genshin/index.html]: 🇵🇱

**Review by hexxcatee — APPROVED** [review]

File: core/gh_webhook_handling.py [hunk 2]
```diff
@@ -46,7 +46,7 @@ def handle_pr(payload, action):
     if action == "opened":
         action = PR.OPENED
     elif action == "closed":
-        if ["pull_request"]["merged"]:
+        if payload["pull_request"]["merged"]:
```
Comment (Norbiros) [inline core/gh_webhook_handling.py]: Update yours branch https://github.com/Gwardia-Czapli/GwardiaHub/commit/6e195e3d307e859a7d8e4b1a9281bf52c429146e

**Review by Norbiros — CHANGES_REQUESTED** [review]

**Review by Norbiros — APPROVED** [review]

**Pull request merged**